### PR TITLE
Polish document content

### PR DESCRIPTION
**Pull request by steverao**
## Changes

Polish content in tracestate-probability-sampling.md

### CODE REVIEW

File: specification/trace/tracestate-probability-sampling.md [hunk 1]
```diff
@@ -222,7 +222,7 @@ incomplete if the span's parent span was not collected.
 Incomplete traces that result from sampling configuration (i.e., on
 purpose) are known as partial traces.  An important subset of the
 partial traces are those which are also complete subtraces.  A
-complete subtrace is defined at span S when every descendent span is
+complete subtrace is defined at span when every descendent span is
```
Comment (MrAlias) [inline specification/trace/tracestate-probability-sampling.md:225]: Can we also fix the incorrect use of "descendent"?

```suggestion
complete subtrace is defined at span when every descendant span is
```

Comment (steverao) [reply specification/trace/tracestate-probability-sampling.md]: > Can we also fix the incorrect use of "descendent"?

Thanks for your remind, I fixed them together now.

**Comment by steverao** [comment]
> Please be sure to update the PR title and description.

done

**Comment by steverao** [comment]
> Please be sure to update the PR title and description.

done.

**Review by tigrannajaryan — APPROVED** [review]

**Comment by tigrannajaryan** [comment]
Editorial. Merging.

**Pull request merged**